CommunityToolkit.sln

```diff
@@ -30,7 +30,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Docs", "Docs", "{88C6FFBE-3
 		CODE_OF_CONDUCT.md = CODE_OF_CONDUCT.md
 		Contributing.md = Contributing.md
 		License.md = License.md
-		README.md = README.md
```

Comment:
Please leave README in uppercase, it's pretty standard.

Reply:
The `README` file without a file extension in uppercase is standard but with a file extension like `.md`, `.txt`, It's usually PascalCase or kebab-case! e.g., `ReadMe.txt` or `Code-of-Conduct.md`. ***Not many people know this.***

The reason I didn't update `CODE_OF_CONDUCT.md` since the updates to the file is managed at org-level. But if you're okay with Normalizing cases then I'll update that file-name too!